image/tarexport/load.go

```diff
@@ -129,7 +134,7 @@ func (l *tarexporter) Load(inTar io.ReadCloser, outStream io.Writer, quiet bool)
 				return fmt.Errorf("invalid tag %q", repoTag)
 			}
 			l.setLoadedTag(ref, imgID.Digest(), outStream)
-			outStream.Write([]byte(fmt.Sprintf("Loaded image: %s\n", reference.FamiliarString(ref))))
+			fmt.Fprintf(outStream, "Loaded image: %s\n", reference.FamiliarString(ref))
```

Comment:
I guess this could also just concatenate (if we want to get rid of `sprintf` / `fprintf`.

```go
outStream.Write([]byte("Loaded image ID: " + reference.FamiliarString(ref) + "\n"))
```

Looks like that's slightly faster

```
BenchmarkWithFprintf2-8   	27867684	        44.11 ns/op	      83 B/op	       0 allocs/op
BenchmarkWithConcat2-8    	29602640	        37.75 ns/op	      49 B/op	       0 allocs/op
```